articles/azure/azs-how-create-S2S-VPN-azurestack-to-azure.md

```diff
@@ -308,7 +312,7 @@ To associate a virtual network with a gateway, it must first contain a valid gat
 
    - **VPN type** - The type of VPN you can choose depends on the make and model of your VPN device, and the kind of VPN connection you intend to create. Choose a route-based gateway if you intend to use point-to-site, inter-virtual network, or multiple site-to-site connections; if you are creating a VPN type gateway to coexist with an ExpressRoute gateway; or if you need to use IKEv2. Policy-based gateways support only IKEv1.
 
-   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. You must select Standard or High performance if you are creating the network to coexist with an ExpressRoute gateway. You must select High performance SKU to enable active-active mode. You can find more information about SKUs here: [Azure Stack Hub SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
+   - **SKU** -Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. From version 1910 onwards, a custom IPSec policy is required for Azure Stack to connect to public Azure, therefore requiring a Standard or High SKU. If you are creating the network to coexist with an ExpressRoute gateway, you must select Standard or High also. You must select High performance SKU to enable active-active mode. You can find more information about SKUs here: [Azure Stack Hub SKUs](https://docs.microsoft.com/en-gb/azure/vpn-gateway/vpn-gateway-about-skus-legacy).
```

Comment:
```suggestion
   - **SKU** - Route-based VPN gateway types are offered in three SKUs: Basic, Standard, and High performance. From version 1910 onwards, a custom IPSec policy is required for Azure Stack Hub to connect to public Azure, therefore requiring a Standard or High performance SKU. If you want to enable active-active mode, you must select High performance SKU.
ExpressRoute gateway types are offered in three SKUs: Standard, High performance, and Ultra performance.
```

Azure Stack Hub only offers the legacy SKUs (Basic, Standard and High performance).

Public Azure offers the Basic legacy SKU, but also offers new VpnGw SKUs. You cannot directly migrate from a legacy SKU to a new SKU.

We need to explain this somehow whilst also referecning the custom IPSec policy.